test/ext/rotbaum/test_rotbaum_smoke.py

```diff
@@ -59,3 +61,69 @@ def test_rotbaum_smoke(datasets):
     predictor = estimator.train(dataset_train)
     forecasts = list(predictor.predict(dataset_test))
     assert len(forecasts) == len(dataset_test)
+
+
+def test_short_history_item_pred():
```

Comment:
@leica2023 would this test fail before the fix contained in the PR? (That is, would it fail if run on the current `dev` branch)

Reply:
correct.